Inseok_45/app.js

```diff
@@ -32,6 +32,37 @@ app.get('/ping', (req, res) => {
     res.status(200).json({ message: "pong" });
 });
 
+app.get('/users/post/view', async (req, res) => {
+    await dataSource.query(
+        `SELECT
+          users.id as userId,
+          posts.id as postingId,
+          posts.post_image as postingImageUrl,
+          posts.post_paragraph as postingContent
+        FROM users
+        INNER JOIN posts ON posts.user_id = users.id`
+            ,(err, rows) => {
+    res.status(200).json({ "data": rows });
+            }
+    );
+})
+
+app.get('/users/post/view/4', async (req, res) => {
```

Comment:
```suggestion
app.get('/users/post/view', async (req, res) => {
```
end point 끝에 `/4`는 왜 들어간걸까요? 중복되면 다른 엔드포인트 이름으로 return값에 맞게 적절히 수정해주면 됩니다!

Reply:
id 가 4인 user의 post 정보를 가져왔다 해서 '4'를 넣었습니다.